### PR TITLE
Render disease cubes

### PR DESCRIPTION
**Pull request by schai6**
Render disease cubes on map.
Added tests.

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -110,3 +110,35 @@ button.ui.toggle.button.player-button + button.ui.toggle.button.player-button {
 button.ui.toggle.button.card-button + button.ui.toggle.button.card-button {
   margin-left: 10px;
 }
+
+/* DiseaseCube */
+img.leaflet-marker-icon.disease-cube-0.leaflet-zoom-animated.leaflet-interactive {
+  margin-left: 5px !important;
```
Comment (sarahzhao25) [inline public/style.css]: Is there no better way to do this besides using the !important tab...

Comment (schai6) [reply public/style.css]: I used !important to overwrite the inline styling created by leaflet. Can you think of another way?

Comment (sarahzhao25) [reply public/style.css]: Oooohhh I see... can you not do inline styling here? & also merge with master plz!

Comment (schai6) [reply public/style.css]: Nope since the Marker leaflet component doesn't take in styling. Unless there's a way to do that?

**Pull request merged**